websocket/include/dlib/socket.h

```diff
@@ -0,0 +1,252 @@
+// Copyright 2020-2022 The Defold Foundation
```

Comment:
I think there was some miscommunication on our Discord regarding what should be exposed or not.

Yes, we need our own extensions to use the proper sdk headers.
During the next release period I will make sure the needed (usually bare minimum) functions are exposeed in a `dmsdk/dlib/socket.h` etc.

Reply:
Thank you, that would be much better.